gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/main/java/io/gravitee/am/gateway/handler/scim/model/User.java

```diff
@@ -286,179 +290,9 @@ public class User extends Resource {
-    }
-
-    public String getTitle() {
-        return title;
-    }
-
-    public void setTitle(String title) {
-        this.title = title;
-    }
-
-    public String getUserType() {
-        return userType;
-    }
-
-    public void setUserType(String userType) {
-        this.userType = userType;
-    }
-
-    public String getPreferredLanguage() {
-        return preferredLanguage;
-    }
-
-    public void setPreferredLanguage(String preferredLanguage) {
-        this.preferredLanguage = preferredLanguage;
-    }
-
-    public String getLocale() {
-        return locale;
-    }
-
-    public void setLocale(String locale) {
-        this.locale = locale;
-    }
-
-    public String getTimezone() {
-        return timezone;
-    }
-
-    public void setTimezone(String timezone) {
-        this.timezone = timezone;
-    }
-
-    public boolean isActive() {
-        return active;
-    }
-
-    public void setActive(boolean active) {
-        this.active = active;
-    }
-
-    public String getPassword() {
-        return password;
-    }
-
-    public void setPassword(String password) {
-        this.password = password;
-    }
-
-    public List<Attribute> getEmails() {
-        return emails;
-    }
-
-    public void setEmails(List<Attribute> emails) {
-        this.emails = emails;
-    }
-
-    public List<Attribute> getPhoneNumbers() {
-        return phoneNumbers;
-    }
-
-    public void setPhoneNumbers(List<Attribute> phoneNumbers) {
-        this.phoneNumbers = phoneNumbers;
-    }
-
-    public List<Attribute> getIms() {
-        return ims;
-    }
-
-    public void setIms(List<Attribute> ims) {
-        this.ims = ims;
-    }
-
-    public List<Attribute> getPhotos() {
-        return photos;
-    }
-
-    public void setPhotos(List<Attribute> photos) {
-        this.photos = photos;
-    }
-
-    public List<Address> getAddresses() {
-        return addresses;
-    }
-
-    public void setAddresses(List<Address> addresses) {
-        this.addresses = addresses;
-    }
-
-    public List<Member> getGroups() {
-        return groups;
-    }
-
-    public void setGroups(List<Member> groups) {
-        this.groups = groups;
-    }
-
-    public List<String> getEntitlements() {
-        return entitlements;
-    }
-
-    public void setEntitlements(List<String> entitlements) {
-        this.entitlements = entitlements;
-    }
-
-    public List<String> getRoles() {
-        return roles;
-    }
-
-    public void setRoles(List<String> roles) {
-        this.roles = roles;
-    }
-
-    public List<Certificate> getX509Certificates() {
-        return x509Certificates;
-    }
-
-    public void setX509Certificates(List<Certificate> x509Certificates) {
-        this.x509Certificates = x509Certificates;
-    }
-
-    public String getSource() {
-        return source;
-    }
+    /**
+     * Extended custom AM source field. Represents last password update.
+     */
+    private Date lastPasswordReset;
```

Comment:
**question**: as it is not a standard SCIM field, can you please ask to product team, if it is OK to have it in this "default" SCIM user model or do we have instead to move it in the GraviteeUser model ?